Library/Homebrew/api.rb

```diff
@@ -15,12 +15,10 @@ module API
 
     extend Cachable
 
-    module_function
```

Comment:
I admittedly don't like `module_fuction`, but I also don't think the functions below actually work as private instance methods, they rely on `extend Cachable`, which only makes methods available at the class level, iiuc.